UI/goliveapi-postdata.cpp

```diff
@@ -66,6 +67,21 @@ constructGoLivePost(QString streamKey,
 		preferences.composition_gpu_index = ovi.adapter;
 	}
 
+	obs_audio_info oai;
+	if (obs_get_audio_info(&oai)) {
+		preferences.audio_samples_per_sec = oai.samples_per_sec;
+		preferences.audio_channels = get_audio_channels(oai.speakers);
+	}
+
+	if (config_get_bool(GetGlobalConfig(), "Audio",
+			    "LowLatencyAudioBuffering")) {
+		preferences.audio_fixed_buffering = true;
+		preferences.audio_max_buffering_ms = 20;
+	} else {
+		preferences.audio_fixed_buffering = false;
+		preferences.audio_max_buffering_ms = 1000;
```

Comment:
For correctness sake, this should depend on the configured sample rate, so perhaps some maths is needed here:
https://github.com/obsproject/obs-studio/blob/7a90c80b36b5805cf671538e2818ee6248683340/libobs/obs.c#L1626-L1628

Reply:
@derrod I'd like to make sure I'm understanding the feedback on this. To query directly from OBS's audio configuration seems to imply either there is an existing public function that returns the required information, or an existing functionn eeds to be modified, or a new function needs to be added. I can't seem to find an existing function that returns the max_buffering or fixed_buffering flag. The closest is the the `obs_audio_info2` struct in obs.h, but this doesn't seem to be stored anywhere with an access function.

If there is an existing function that returns the necessary info, please let me know what that is. Thanks.